floss/main.py

```diff
@@ -1015,8 +1052,8 @@ def main(argv=None):
         return 1
 
     try:
-        vw = load_vw(sample_file_path, options.save_workspace, options.verbose, options.is_shellcode,
-                     options.shellcode_entry_point, options.shellcode_base)
+        vw = load_vw(sample_file_path, complementary_idb_file_path, options.save_workspace, options.verbose, options.is_shellcode,
```

Comment:
this line is getting long, lets reformat it

Reply:
https://github.com/BenjaminSoelberg/flare-floss/commit/3dceb593f36123530e91007b410ca56502f18854